source/Engine.cpp

```diff
@@ -246,7 +246,9 @@ Engine::Engine(PlayerInfo &player)
 	isMouseToggleEnabled(Preferences::Has("alt-mouse turning")),
 	ammoDisplay(player), shipCollisions(256u, 32u)
 {
-	zoom = Preferences::ViewZoom();
+	baseZoom = Preferences::ViewZoom();
+	zoomMod = Preferences::Has("Cinematic effects") ? 2. : 1.;
```

Comment:
Looks like you're still using the old preference name here.

Reply:
Thanks for the catch-on